modules/post/windows/gather/credentials/flock.rb

```diff
@@ -1,12 +1,9 @@
-# frozen_string_literal: true
```

Comment:
I think the frozen literals are fine here; is rubocop removing them or was this manual? 👀

Reply:
These were removed manually.